addons/hr_holidays/static/src/bugfix/bugfix.js

```diff
@@ -8,3 +8,77 @@ odoo.define('hr_holidays/static/src/bugfix/bugfix.js', function (require) {
 'use strict';
 
 });
+
+// FIXME move me in hr_holidays/static/src/models/partner/partner.js
+odoo.define('hr_holidays/static/src/models/partner/partner.js', function (require) {
+'use strict';
+
+const {
+    registerClassPatchModel,
+    registerFieldPatchModel,
+    registerInstancePatchModel,
+} = require('mail/static/src/model/model_core.js');
+const { attr, one2one } = require('mail/static/src/model/model_field.js');
+
+const { str_to_datetime } = require('web.time');
+
+
+registerClassPatchModel('mail.partner', 'hr_holidays/static/src/models/partner/partner.js', {
+    /**
+     * @override
+     */
+    convertData(data) {
+        const data2 = this._super(data);
+        if ('out_of_office_date_end' in data) {
+            data2.out_of_office_date_end = data.out_of_office_date_end;
+        }
+        return data2;
+    },
+});
+
+registerInstancePatchModel('mail.partner', 'hr_holidays/static/src/models/partner/partner.js', {
+    /**
+     *
+     * @private
+     */
+    _computeOutOfOfficeText() {
+        if (!this.out_of_office_date_end) {
+            return;
+        }
+        if (!this.messaging.locale.language) {
+            return;
+        }
+        const currentDate = new Date();
+        const date = str_to_datetime(this.out_of_office_date_end);
+        const options = { day: 'numeric', month: 'short' };
+        if (currentDate.getFullYear() !== date.getFullYear()) {
+            options.year = 'numeric';
+        }
+        const localeCode = this.messaging.locale.language.replace(/_/g,'-');
+        const formattedDate = date.toLocaleDateString(localeCode, options);
+        return _.str.sprintf(this.env._t("Out of office until %s"), formattedDate);
```

Comment:
need to add new/moved string in `pot` file

also before PR there was a dot at the end

Reply:
Yes for the point but the formatted date already has a dot... so it brings two dots in the sentence

Reply:
How do I add it ? I tried to regen the .pot file for hr_holidays but it's not in it

Reply:
How do I add this in .pot ? I tried to regenerate from interface but it does not add that line

Reply:
Manually I guess, just follow the format of others in the file.

Reply:
Now that I think about it, I think I have seen a PR where files not in `static/src/js` specifically were ignored?
Do you remember @alexkuhn?

Reply:
Yes, there's a PR in progress to fix that in v14, so that every file in `static/src` is eligible for translation.
FYI https://github.com/odoo/odoo/pull/61809